public/app/plugins/datasource/cloudwatch/datasource.ts

```diff
@@ -510,6 +511,7 @@ export class CloudWatchDatasource
     const requestParams: GetLogEventsRequest = {
       limit,
       startFromHead: direction !== 'BACKWARD',
+      region: query?.region,
```

Comment:
is there ever a situation where region is not defined? Will it still work? I could see a situation where you have to be like `query?.region || "__default"` or something like that, but I'm not 100% sure that's necessary, just a question to look into